test/e2e-common.sh

```diff
@@ -162,6 +162,42 @@ function gateway_conformance() {
   popd
 }
 
+function test_conformance_no_gw_service() {
```

Comment:
can probably just call this `test_e2e` - since it's running our e2e tests in this repo